lib/logstash/outputs/elasticsearch.rb

```diff
@@ -263,6 +265,10 @@ class LogStash::Outputs::ElasticSearch < LogStash::Outputs::Base
   config :user, :validate => :string
   config :password, :validate => :password
 
+  # HTTP Path at which the Elasticsearch server lives. Use this if you must run ES behind a proxy that remaps
+  # the root path for the Elasticsearch HTTP API lives. This option is ignored for non-HTTP transports.
+  config :path, :validate => :string, :default => "/"
```

Comment:
It might be inconvenient if someone's just trying out one transport vs. the other. Could be handy to leave it in? I can remove it if you think that doesn't make sense however.